ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,156 +1,166 @@
-
-{%             endfor %}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.enabled is defined and vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.enabled == true %}
-   multicast ipv4 source route export{% if vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance is defined and vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance is not none %} {{ vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance }}{% endif %}
-
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].access_group_in is defined and vlan_interfaces[vlan_interface].access_group_in is not none %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ipv6_nd_prefixes is arista.avd.defined %}
+{%         for prefix in vlan_interfaces[vlan_interface].ipv6_nd_prefixes %}
+{%             set ipv6_nd_prefix_cli = "ipv6 nd prefix " ~ prefix %}
+{%             if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].valid_lifetime is arista.avd.defined %}
+{%                 set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ " " ~ vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].valid_lifetime %}
+{%             endif %}
+{%             if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].preferred_lifetime is arista.avd.defined %}
+{%                 set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ " " ~ vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].preferred_lifetime %}
+{%             endif %}
+{%             if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].no_autoconfig_flag is arista.avd.defined(true) %}
+{%                 set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ " no-autoconfig" %}
+{%             endif %}
+   {{ ipv6_nd_prefix_cli }}
+{%         endfor %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.enabled is arista.avd.defined(true) %}
+{%         if vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance is arista.avd.defined %}
+   multicast ipv4 source route export {{ vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance | arista.avd.default('') }}
```

Comment:
Do we need an else, if we set a default of '' - nothing. Because the if condition on line 83 fulfils both cases right?

As if you set nothing it will still print no admin distance?